packages/visualizations/src/components/Map/WebGl/MapRender.svelte

```diff
@@ -149,12 +153,17 @@
         resizer = new ResizeObserver(
             debounce(() => {
                 map.resize();
-                if (mapReady && viewBox) {
-                    setViewBox(viewBox);
+                if (isViewBoxEqualToMap) {
+                    map.setMaxBounds(null);
```

Comment:
I have a hard time reproducing a broken case with my data, but the `isViewBoxEqualToMap` means situations where we don't pass any viewBox, and it defaults to bbox right? Why is this situation special compared to specific viewbox and bbox? Maybe is it the other way around, and we only want to do this maxbounds gymnastic if there is already one (`map.getMaxBounds()`) at the moment of the resize?

Reply:
Although that would be dangerous as well because you could have a bbox and then maxBounds that is very wide, and it would be reduced to the current viewBox suddenly, so no my logic is wrong 🙈

Reply:
Any specific reason why we couldn't just setMaxBounds to the `bbox` option directly? There has to be a reason we didn't do this at the start, but I'm not sure why anymore. If so this would probably remove the need for all that.

Reply:
I wasn't precise enough on my PR description, it's because setting maxBounds is not only related to the bbox but more about the map canvas size... For example same bbox but different map aspect ratio results in different maxBounds

Reply:
I find the double conditions on `isViewBoxEqualToMap` a bit strange too.

IMO, we should not have to deal with maxBounds resizing, maxbound are the same whatever the zoom is. In my testing just:
```suggestion
                if (mapReady && currentViewBox) {
                    setViewBox(currentViewBox);
                }
```
seems to work unless I'm missing a case. We just fit the view on resize.

This was more something I forgot to remove in the first place with the introduction of the viewbox I think.